Schutzfile

```diff
@@ -289,14 +289,14 @@
   "centos-10": {
     "dependencies": {
       "osbuild": {
-        "commit": "a6f935e9393b25c7b482afbc8ddf1226114a2289"
+        "commit": "50a6f2733ef2c4d8081020c9d7794713a17d46dc"
       }
     }
   },
   "centos-stream-10": {
     "dependencies": {
       "osbuild": {
-        "commit": "a6f935e9393b25c7b482afbc8ddf1226114a2289"
+        "commit": "50a6f2733ef2c4d8081020c9d7794713a17d46dc"
```

Comment:
You need to use the previous commit. This one points to the post-release version bump, which specifies `[skip ci]` and as a result, no RPMs are built for it. And so there are no RPMs to install and all tests fail due to the broken dependency of composer that is being tested :cry:

Reply:
Oh right.  I keep forgetting that!
Thanks

Reply:
Fixed.